### PR TITLE
Fix overfit option, (overfit lr scheduler patience 100, min_lr=2e-3), cleaner output

### PR DESCRIPTION
**Pull request by kyrylogr**

### CODE REVIEW

File: training/train.py [hunk 1]
```diff
@@ -36,38 +36,46 @@
...
     mode="min",
-    factor=0.2,
-    patience=7,
+    factor=0.5,
+    patience=patience,
     threshold=1e-4,
     threshold_mode="rel",
     cooldown=1,
-    min_lr=1e-6,
+    min_lr=min_lr,
```
Comment (bubanoid) [inline training/train.py:78]: I believe the purpose of the overfit mode is to determine the learning rate scheduler parameters that enable the fastest model training. Therefore, we will use the same parameters for training the model in non-overfit mode as well. However, I am not sure about it. @bulygin1985 please correct me.

Comment (kyrylogr) [reply training/train.py:78]: The purpose of overfit mode is to prove that your model design works - network is able to train, and give expected results in the end. 
There probably can be difference in learning rate for overfit and real training scenario, as overfit batch is 10 and real batches in training will something like 128 or 256 or whatever GPU RAM will allow.

File: training/train.py [hunk 2]
```diff
@@ -36,38 +36,46 @@
...
+else:
+    tag = ""
+    min_lr = 1e-5
+    patience = 7
+    stop_loss = None
+    stop_epoch = 500
+
+
+def criteria_satisfied(current_loss, current_epoch):
+    if stop_loss is not None and current_loss < 1.0:
```
[the remark this answers is not in the document]
Comment (kyrylogr) [reply training/train.py:57]: You can set separate criteria for different modes via different stop_loss or stop_epoch values.
I personally see no problem in having two conditions working together (both stop_epoch or stop_loss being not null).

File: models/checkpoints/readme.txt [hunk 3]
```diff
@@ -1 +1,5 @@
-It is a placeholder for pretrained model weights
+It is a placeholder for pretrained model weights.
+
+You can load save model weights vith the following python code (for cuda execution):
```
[the remark this answers is not in the document]
Comment (kyrylogr) [reply models/checkpoints/readme.txt]: fixed

File: training/train.py (hunk 1 above)
Comment (bulygin1985) [reply training/train.py:79]: @kyrylogr description is closer to the truth. The overfit learning rate scheduler could differ from the training mode learning rate scheduler. The reason is different batch size between overfit and training modes. Roughly, multiplied batch size at "n" time leads to multiplied learning rate at "n" times. See: https://arxiv.org/pdf/1706.02677 , "2.1. Learning Rates for Large Minibatches"

**Review by bulygin1985 — APPROVED** [review]
